code/api/src/modules/user/resolvers.js

```diff
@@ -7,6 +7,28 @@ import serverConfig from '../../config/server'
 import params from '../../config/params'
 import models from '../../setup/models'
 
+// Update
+export async function update(parentValue, { id, survey, style }, { auth }) {
+  style = JSON.parse(style);
+  var user = models.User.findOne({ where: { id } })
+
+  const vals = Object.values(style);
+  const styleChoice = Math.max(...vals);
+  const key = Object.keys(style)[Object.values(style).indexOf(styleChoice)];
```

Comment:
I think this line could have been also been written the following way right?
`const keys = Object.keys(style)[vals.indexOfStyleChicce]`